### PR TITLE
Remove tiles from Community Service

### PR DESCRIPTION
**Pull request by ajcoppa**
Took these changes from @plasticmacaroni's PR #109 but on a fresh branch so we didn't have to resolve some conflicts that cropped up since it was submitted. I've confirmed they work and remove all the annoying reminders that aren't real in CS.

> Updates for Community Service only:
>
>- Removes Continuum Transfunctioner tile
>- Removes Daily Dungeon reminder tile
>- Removes Gelatinous Cubeling reminder tile
>- Does not recommend free Goofballs (can't get there)
>- Does not recommend free science tentacle fight (Summon skill free fight remains)
>
>Other:
>- Tried to fix a random spelling error on a joke from years ago - "autumnaically" -> "autumn-atically"

### CODE REVIEW

File: Source/relay/TourGuide/Sets/Daily Dungeon.ash [hunk 1]
```diff
@@ -14,7 +14,10 @@ void LockPickingGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry
 
 void SDailyDungeonGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [int] optional_task_entries, ChecklistEntry [int] future_task_entries)
 {
-	
+	if (my_path().id == PATH_COMMUNITY_SERVICE) {
```
Comment (docrostov) [inline Source/relay/TourGuide/Sets/Daily Dungeon.ash:17]: while we're here, are there any other paths we can return the tile on? ed the undying is one, i feel like there's 1-2 others but they aren't coming to mind. maybe we just add actually ed here and call it a day.

**Review by docrostov — DISMISSED** [review]
only one small change (add ed to the dungeon tile return) but otherwise looks good, merge at your leisure

File: Source/relay/TourGuide/Sets/Daily Dungeon.ash (hunk 1 above)
Comment (ajcoppa) [reply Source/relay/TourGuide/Sets/Daily Dungeon.ash]: Added Ed to the conditional too. I skimmed a bunch of other paths' wiki pages and didn't find any others, but easy enough to add later if we find more.

**Review by docrostov — APPROVED** [review]
looks good, re-approving and merging

**Pull request merged**